libstdc++-v3/config.h.in

```diff
@@ -766,6 +766,9 @@
    namespace std for C++11. */
 #undef _GLIBCXX_USE_C99_MATH_FUNCS
 
+/* Define if C99 long double math functions are available in <math.h>. */
+#undef _GLIBCXX_USE_C99_MATH_LONG_DOUBLE_FUNCS
```

Comment:
We don't actually need this change any longer as the only C library we support is picolibc, which has all of the long double math functions.

Reply:
@keith-packard IIRC, x86 was the problematic one.

> [It looks like glibc (and hence libgcc) doesn't actually have soft float support for 80-bit values. Looks like we either need to fix that, or bail on long double support for -msoft-float 32-bit x86 targets.](https://github.com/zephyrproject-rtos/sdk-ng/issues/593#issuecomment-1383230197)

I assume this has not changed since then?

Reply:
Nope, still a gap afaict. Sorry for the noise.